src/main.ts

```diff
@@ -78,17 +83,20 @@ async function runForReleaseEvent(vt: VirusTotal) {
         await vt.monitorItems(await github.downloadReleaseAsset(octokit, asset, path.join(context.tmpDir(), asset.name)), inputs.monitorPath).then(upload => {
           outputAnalysis.push(`${asset.name}=${upload.url}`);
           core.info(`${asset.name} successfully uploaded. Check detection analysis at ${upload.url}`);
-          release.body = release.body.concat(`\n* [\`${asset.name}\`](${upload.url})`);
+          release.body = release.body.concat(`\n  * [\`${asset.name}\`](${upload.url})`);
         });
       } else {
         await vt.files(await github.downloadReleaseAsset(octokit, asset, path.join(context.tmpDir(), asset.name))).then(upload => {
           outputAnalysis.push(`${asset.name}=${upload.url}`);
           core.info(`${asset.name} successfully uploaded. Check detection analysis at ${upload.url}`);
-          release.body = release.body.concat(`\n* [\`${asset.name}\`](${upload.url})`);
+          release.body = release.body.concat(`\n  * [\`${asset.name}\`](${upload.url})`);
         });
       }
     });
   });
+  if (assets.length > 5) {
+    release.body = release.body.concat(`\n\ncreated by [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</details>\n`);
```

Comment:
There is an extra newline that doesn't look quite nice:

```
<details>
  <summary>🛡 VirusTotal analysis</summary>

  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)

created by [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)
</details>
```

![image](https://user-images.githubusercontent.com/1951866/174490105-cb0ac5b9-121f-4ceb-906e-6b89256db82e.png)

___

I think it looks better with a `<sub>` like this instead:

```
<details>
  <summary>🛡 VirusTotal analysis</summary>
  
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
  * [`discord-portable-win32.exe`](https://www.virustotal.com/gui/file-analysis/NzMwMTgxNTk3MjMyNjk1NTUyNDI5MjAxMTIxMjllYzk6MTY0NjYwNjQ5Mg==/detection)
<sub>These scans were produced by the [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</sub>
</details>
```

```suggestion
    release.body = release.body.concat(`\n<sub>These scans were produced by the [VirusTotal GitHub Action](https://github.com/crazy-max/ghaction-virustotal)</sub></details>\n`);
```

![image](https://user-images.githubusercontent.com/1951866/174490075-f3211f1d-1c00-4c90-9830-b8cab0ae04e2.png)